run-dask-process.sh

```diff
@@ -134,7 +129,7 @@ function buildUCXWithInfinibandArgs {
 }
```

Comment:
We need to add to `WORKER_ARGS `

`--rmm-async `

Reply:
Thanks, fixed in [de23228](https://github.com/rapidsai/multi-gpu-tools/pull/24/commits/de232283fa27014b41b2241f34b2575e5299204a).